yoti-sdk-api/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java

```diff
@@ -7,31 +7,36 @@
 import com.fasterxml.jackson.annotation.JsonProperty;
 
 public final class Receipt {
-    @JsonProperty("receipt_id")
+
+    @JsonProperty(Property.RECEIPT_ID)
     private byte[] receiptId;
-    @JsonProperty("other_party_profile_content")
+    @JsonProperty(Property.OTHER_PARTY_PROFILE_CONTENT)
     private byte[] otherPartyProfile;
-    @JsonProperty("profile_content")
+    @JsonProperty(Property.PROFILE_CONTENT)
     private byte[] profile;
-    @JsonProperty("other_party_extra_data_content")
+    @JsonProperty(Property.OTHER_PARTY_EXTRA_DATA_CONTENT)
     private byte[] otherPartyExtraData;
-    @JsonProperty("extra_data_content")
+    @JsonProperty(Property.EXTRA_DATA_CONTENT)
     private byte[] extraData;
-    @JsonProperty("wrapped_receipt_key")
+    @JsonProperty(Property.WRAPPED_RECEIPT_KEY)
     private byte[] wrappedReceiptKey;
-    @JsonProperty("policy_uri")
+    @JsonProperty(Property.POLICY_URI)
     private String policyUri;
-    @JsonProperty("personal_key")
+    @JsonProperty(Property.PERSONAL_KEY)
     private byte[] personalKey;
-    @JsonProperty("remember_me_id")
+    @JsonProperty(Property.REMEMBER_ME_ID)
     private byte[] rememberMeId;
-    @JsonProperty("parent_remember_me_id")
+    @JsonProperty(Property.PARENT_REMEMBER_ME_ID)
     private byte[] parentRememberMeId;
-    @JsonProperty("sharing_outcome")
+    @JsonProperty(Property.SHARING_OUTCOME)
     private Outcome outcome;
-    @JsonProperty("timestamp")
+    @JsonProperty(Property.TIMESTAMP)
     private String timestamp;
 
+    public boolean hasSuccessOutcome() {
+        return Outcome.SUCCESS == outcome;
+    }
```

Comment:
Would be better has:
```java
public boolean hasOutcome(Outcome outcome) {
    return this.outcome == outcome;
}
```
for a couple of reasons:
- Even if now there's only two outcomes in the future could be more 
- The code that calls this method becomes more readable (and flexible) if it's able to check for either success or failure (or whatever other outcomes may exist in the future)
- Hardcoding state (the value of an enum) into a method name is best avoided as it creates unnecessary coupling between the code and state.